### ISSUE TITLE
On KDE Plasma, touchpad settings do not work after restart.

### ISSUE THREAD

**Issue by yerbestpal** [clearlinux/distribution#874]
When I try to adjust them again, I receive the error `Active settings don't match saved settings.` 
![clear2](https://user-images.githubusercontent.com/15250241/59268731-45037f80-8c45-11e9-95d4-086cdc467659.png)
Clicking the 'show active settings' button allows me to adjust those settings, but not apply them. This means that when I close the window, they are lost.

**Comment by anselmolsm** [comment]
@yerbestpal is it a wayland session?

**Comment by yerbestpal** [comment]
@anselmolsm Nope. In fact the Wayland session just crashes as soon as I try to log in.

**Comment by tanrax** [comment]
Indeed, when I restart my settings, they have no effect. I have to change some option and save for it to be applied.
